rareapi/views/categories.py

```diff
@@ -1,14 +1,53 @@
 """View module for handling requests about game types"""
+from django.core.exceptions import ValidationError
+from rest_framework import status
 from django.http import HttpResponseServerError
 from rest_framework.viewsets import ViewSet
 from rest_framework.response import Response
 from rest_framework import serializers
-from rareapi.models import Categories
+from rareapi.models import Categories, categories
 
 
 class CategoriesViewset(ViewSet):
     """rare restful Categories"""
 
+    def create(self, request):
+        """Handle POST operations
+
+        Returns:
+            Response -- JSON serialized category instance
+        """
+
+        # Uses the token passed in the `Authorization` header
+        categories = Categories.objects.get(user=request.auth.user)
+
+        # Create a new Python instance of the Game class
+        # and set its properties from what was sent in the
+        # body of the request from the client.
+        category = Category()
```

Comment:
This needs to be `category = Categories()` since your model name is `Categories`